code.swift

```diff
@@ -54,25 +54,24 @@ class PeriodicTableViewController: UIViewController {
     }
     
     // This next method was added by a new member of the team. Identify the 10 formatting, naming, and logic issues below:
-    @IBAction func chooseRandomElement(_ sender: Any){
-        let ranElem = elementsContoller.elements.randomElement()
+    @IBAction func chooseRandomElement(_ sender: Any) {
+        let ranElem = elementsContoller.elements
         
-        var chosenElementView:ElementView!
+        var chosenElementView: ElementView
         
-        for e in elementViews {
-            if (e.element == ranElem) {
```

Comment:
You are maybe the only one that found the formatting issue here, but you also changed a property when you renamed `e` that didn’t need to be changed... what mistake do you think you made?